nvflare/tool/code_pre_installer/README.md

```diff
@@ -0,0 +1,119 @@
+# NVFLARE Code Pre-Installer
+
+A tool to pre-install NVFLARE application code and libraries.
+
+## Overview
+
+The code pre-installer handles:
+- Installation of application code
+- Installation of shared libraries
+- Site-specific customizations
+- Python package dependencies
+
+## Directory Structure
+
+Expected application code zip structure:
+```
+app_code.zip
+├── app_code/
+│   ├── meta.json           # Application metadata
+│   ├── apps/              # Default application code
+│   │   └── custom/        # Default custom code
+│   └── app_site-1/        # Site-specific code (optional)
+│       └── custom/        # Site custom code
```

Comment:
I don't see why we would need this?
Should we just use one image for each site?